<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushOpen">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRate">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBits">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="parity">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBits">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>BaudRate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Bits:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Bits:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="portList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="terminal_write">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Response</string>
   </property>
  </widget>
  <widget class="QPushButton" name="terminal_w_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_closeTerminal">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_ports">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="terminal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>661</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="portState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanTerminal">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CR_cbox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="LF_cBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>LF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
